doc/man3/OSSL_HTTP_transfer.pod

```diff
@@ -0,0 +1,210 @@
+=pod
+
+=head1 NAME
+
+OSSL_HTTP_get,
+OSSL_HTTP_get_asn1,
+OSSL_HTTP_post_asn1,
+OSSL_HTTP_transfer,
+OSSL_HTTP_bio_cb_t,
+OSSL_HTTP_proxy_connect,
+OSSL_HTTP_parse_url
+- http client functions
+
+=head1 SYNOPSIS
+
+ #include <openssl/http.h>
+
+ typedef BIO *(*OSSL_HTTP_bio_cb_t)(BIO *bio, void *arg,
+                                    int connect, int detail);
+ BIO *OSSL_HTTP_get(const char *url, const char *proxy, const char *proxy_port,
+                    BIO *wbio, BIO *rbio,
+                    OSSL_HTTP_bio_cb_t bio_update_fn, void *arg,
+                    const STACK_OF(CONF_VALUE) *headers,
+                    int maxline, unsigned long max_resp_len, int timeout,
+                    const char *expected_content_type, int expect_asn1);
+ ASN1_VALUE *OSSL_HTTP_get_asn1(const char *url,
+                                const char *proxy, const char *proxy_port,
+                                BIO *wbio, BIO *rbio,
```

Comment:
I don't know how important it is, but I observe that with `SSL_set_bio` we pass the bios in the order rbio, wbio. But here we do it the other way around. Should we consistently do it the same?

Reply:
While I see the consistency point, here I suggest to keep the current order because 
* the client uses the first mentioned BIO (whose name may change, see my last comment) first: the client uses it for connecting and/or for writing requests before optionally using `rbio` for reading responses
* if the first BIO is NULL the second must be NULL, too (but not the other way round), so the first BIO is used in more use cases.

Reply:
Due to renaming `wbio` to  `bio`, which indicates that it may be of more broad use, in my view the consistency point has become even weaker. 
So in my view the current order of parameters is adequate.